docs/website/src/main/resources/certifications/jakarta-xml-registries/1.1/TCK-Results.md

```diff
@@ -0,0 +1,35 @@
+TCK Results
+===========
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta XML Registries.
+
+# Eclipse GlassFish 5.1, Full Profile, Certification Request
+
+- Product Name, Version and download URL (if applicable) \
+  [Eclipse GlassFish 5.1](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
+  Includes XML Registries 1.0.9
```

Comment:
These results should be hosted by the [jaxr-ri](https://github.com/eclipse-ee4j/jaxr-ri) project.

Reply:
Links back to jaxr-ri project. Project may use this, or copy and we can delete.